Gems/Atom/RPI/Code/Include/Atom/RPI.Public/Pass/ComputePass.h

```diff
@@ -78,9 +78,18 @@ namespace AZ
             // The draw item submitted by this pass
             RHI::DispatchItem m_dispatchItem;
 
-            // Whether or not to make the pass a full-screen compute pass. If set to true, the dispatch group counts will
-            // be automatically calculated from the size of the first output attachment and the group size dimensions.
-            bool m_isFullscreenPass = false;
+            // Whether or not to make the pass a full-screen compute pass, and which slot to use as the size source
+            AZ::Name m_fullscreenSizeSourceSlot;
+            bool m_fullscreenDispatch = false;
+            AZ::RPI::PassAttachmentBinding* m_fullscreenSizeSourceBinding = nullptr;
+
+            // Wheter or not to make the pass a indirect dispatch compute pass, and which slot to use as indirect dispatch
+            // command buffer
+            bool m_indirectDispatch = false;
+            AZ::Name m_indirectDispatchBufferSlot;
```

Comment:
Can you change the variable name from m_indirectDispatchBufferSlot to m_indirectDispatchBufferSlotName to make it explicit that it's the name and not the slot?
Later in the code you have m_indirectDispatchBufferSlot.GetCStr() which is confusing, would be easier to understand with m_indirectDispatchBufferSlotName.GetCStr()